components/login/login_controller/login_controller.jsx

```diff
@@ -364,11 +364,13 @@ class LoginController extends React.Component {
         if (this.props.enableCustomBrand) {
             const text = this.props.customBrandText || '';
             const formattedText = TextFormatting.formatText(text);
+            const brandImageUrl = Client4.getBrandImageUrl(0);
 
             return (
                 <div>
-                    <img
-                        src={Client4.getBrandImageUrl(0)}
+                    <object
+                        data={brandImageUrl}
+                        type='image/jpg'
```

Comment:
Are we sure this will always be a jpeg? I'm not sure if we make any changes to the brand image to make it a consistent type